components/encryptVideos.js

```diff
@@ -0,0 +1,21 @@
+module.exports = [
+      {
```

Comment:
Do you mind moving this to a directory call `data` or something since this is now a data file rather than a component.